lib/ui-message-queue.js

```diff
@@ -72,32 +83,42 @@
 
   })();
 
-  /*
-  UiMessageQueue - a User Message Queue Javascript implementation
-  (c) 2014 Rob Murray
-  @see https://github.com/rob-murray/ui-message-queue
-  UiMessageQueue may be freely distributed under the MIT license
-  */
+  root = typeof exports !== "undefined" && exports !== null ? exports : window;
+
+  root.FifoQueue = FifoQueue;
+
+}).call(this);
+
 
+/*
+UiMessageQueue - a User Message Queue Javascript implementation
+(c) 2014 Rob Murray
+@see https://github.com/rob-murray/ui-message-queue
+UiMessageQueue may be freely distributed under the MIT license
+ */
+
+(function() {
+  var AlertDisplay, DomDisplay, FifoQueue, UiMessageQueue, root,
+    bind = function(fn, me){ return function(){ return fn.apply(me, arguments); }; };
 
   root = typeof exports !== "undefined" && exports !== null ? exports : window;
 
-  AlertDisplay = root.AlertDisplay || require('../lib/AlertDisplay').AlertDisplay;
+  FifoQueue = root.FifoQueue || require('../src/FifoQueue').FifoQueue;
 
-  DomDisplay = root.DomDisplay || require('../lib/DomDisplay').DomDisplay;
+  DomDisplay = root.DomDisplay || require('../src/DomDisplay').DomDisplay;
 
-  FifoQueue = root.FifoQueue || require('../lib/FifoQueue').FifoQueue;
+  AlertDisplay = root.AlertDisplay || require('../src/AlertDisplay').AlertDisplay;
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'require' is not defined.